infra/modules/azure_naming_convention/locals.tf

```diff
@@ -0,0 +1,5 @@
+locals {
+  location_short       = var.environment.location == "italynorth" ? "itn" : var.environment.location == "westeurope" ? "weu" : var.environment.location == "germanywestcentral" ? "gwc" : "neu"
```

Comment:
I'd rather not default and use a map, something alike:

```
locals {
  location_map = {
    "italynorth" : "itn",
    "westeurope" : "weu",
    "germanywestcentral" : "gwc"
  }
  location_short = lookup(location_map, var.environment.location, "Error: Invalid location")

  project              = "${var.environment.prefix}-${var.environment.env_short}-${local.location_short}"
  resource_name_prefix = "${local.project}-${var.environment.domain}-${var.environment.app_name}"
}

```

Reply:
we may centralize this map someway

Reply:
Indeed a better solution. The code is already centralized in the module `azure_naming_convention`